deploy/helm/node-feature-discovery-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: {{ include "node-feature-discovery-operator.fullname" . }}-controller-manager
+  namespace: {{ include "node-feature-discovery-operator.namespace" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      {{- include "node-feature-discovery-operator.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "node-feature-discovery-operator.labels" . | nindent 8 }}
```

Comment:
I think you want to use the selectorLabels here (as you use them in the selector above)

Reply:
I will